### ISSUE TITLE
error: "No overload for method GetEnvironmentVariable

### ISSUE THREAD

**Issue by diemildefreude** [patricksebastien/libpd4unity#13]
http://i64.tinypic.com/ws93k2.png

Slayther: Which is strange, because when I checked the code, it indeed can take 2 arguments

http://i67.tinypic.com/nnlf15.png

Slayther: The second error is also odd

http://i64.tinypic.com/dbtd86.png

**Comment by thefuntastic** [comment]
These methods are specific to the windows build platform. That means if windows isn't selected as your build platform unity won't be able to find these methods. 

Please check in your build settings that the platform is set to PC with Platform > Windows and Architecture > x86.

**Comment by diemildefreude** [comment]
Thanks for the reply. Does this mean that Unity projects made using libpd4unity can't be built to be playable on Mac?

**Comment by thefuntastic** [comment]
No, libpd4unity still works on OSX. This code was just a workaround that meant you didn't have to copy the dll to the unity editor folder on windows.
